### ISSUE TITLE
"traffic_signs_yolov3.h5" & "backend.h5"

### ISSUE THREAD

**Issue by anujsindgi** [AnirudhRajhgopalann/German-Traffic-Sign-Detection#1]
Hey Anirudh, could you please mention how you obtained the traffic_signs_yolov3.h5 & backend.h5 files?

Thanks a lot!

**Comment by anirudh-s97** [comment]
Hey Sorry for the Delayed Reply! 

- traffic_signs_yolov3.h5 is the weights that I trained the model with colab
- And backend.h5 is the weight of the pretrained darknet model
Sorry I'll update this repo with the weights soon!

**Comment by anujsindgi** [comment]
Thank you, I eventually understood it as I read your code. If you would work more on it in future, I'd suggest you also publish results of your model on test set of GTSDB.
